### PR TITLE
Fix SandboxInfo

### PR DESCRIPTION
**Pull request by MichaelDeBoey**
Follow-up of #2961

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/2967/commits/a6a71bd094ee4be53b4d95891613dee0a9d063aa is at https://pr2967.build.csb.dev/s/new.

File: packages/app/src/app/pages/Profile/Showcase/SandboxInfo/index.tsx [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
 } from './elements';
+import SvgButton from './play-button.svg';
+
+type Props = {
+  sandbox: Sandbox;
+};
+export const SandboxInfo: FunctionComponent<Props> = ({ sandbox }) => {
+  const {
+    state: { isLoggedIn },
+  } = useOvermind();
```
Comment (CompuIves) [inline packages/app/src/app/pages/Profile/Showcase/SandboxInfo/index.tsx]: Hmm, I do like the fact that this component is "standalone" without the store, so I'd prefer to keep `useOvermind` out if it's not really needed.

File: packages/app/src/app/pages/Profile/Showcase/SandboxInfo/elements.ts [hunk 2]
```diff
@@ -18,7 +19,6 @@ export const Container = styled.div`
 `;
 
 export const Title = styled.h1`
-  font-weight: 400;
```
Comment (CompuIves) [inline packages/app/src/app/pages/Profile/Showcase/SandboxInfo/elements.ts]: This one wasn't needed?

Comment (MichaelDeBoey) [reply packages/app/src/app/pages/Profile/Showcase/SandboxInfo/elements.ts]: A bit further down we have

```css
font-weight: 300;
```

so this one's obsolete

File: packages/app/src/app/pages/Profile/Showcase/SandboxInfo/index.tsx (hunk 1 above)
Comment (MichaelDeBoey) [reply packages/app/src/app/pages/Profile/Showcase/SandboxInfo/index.tsx]: This component is only used once _and_ we've done a lot of refactors where we used `useOvermind` as near as possible to the needing code.

**Comment by MichaelDeBoey** [comment]
Ping @CompuIves

**Pull request merged**